### PR TITLE
Kinect2 urdfs for PR2

### PR DESCRIPTION
**Pull request by archielee**
- Added Kinect2 urdfs and corresponding mesh
- Included argument passing from launch files so users can choose which Kinect to use on the PR2

### CODE REVIEW

File: pr2_description/robots/pr2.urdf.xacro [hunk 1]
```diff
@@ -24,7 +24,6 @@
   <xacro:include filename="$(find pr2_description)/urdf/base_v0/base.urdf.xacro" />
   <!-- Head sensors -->
   <xacro:include filename="$(find pr2_description)/urdf/sensors/head_sensor_package.urdf.xacro" />
-  <xacro:include filename="$(find pr2_description)/urdf/sensors/kinect_prosilica_camera.urdf.xacro" />
```
Comment (UltronDestroyer) [inline pr2_description/robots/pr2.urdf.xacro:27]: Does this affect anything? It was only the prosilica attached to the kinect and not the prosilica camera in the head, yeah?

File: pr2_description/robots/pr2.urdf.xacro [hunk 2]
```diff
@@ -154,4 +135,36 @@
     <origin xyz=".135 0 .044" rpy="${M_PI/2} ${-32.25*M_PI/180} 0" />
   </xacro:wge100_camera_v0>
 
+  <!-- Kinect models -->
+  <xacro:arg name="kinect2" default="false"/>
+
+  <!-- Kinect1 xacro -->
+  <xacro:unless value="$(arg kinect2)">
+    <xacro:include filename="$(find pr2_description)/urdf/sensors/kinect_prosilica_camera.urdf.xacro" />
```
Comment (UltronDestroyer) [inline pr2_description/robots/pr2.urdf.xacro:143]: Can you change the name of this to reflect that it no longer is about the prosilica camera, but only about the kinect 2?

Edit: nevermind. This is if the kinect2 isn't included.

File: pr2_description/urdf/sensors/iai_kinect2.urdf.xacro [hunk 3]
```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
```
Comment (UltronDestroyer) [inline pr2_description/urdf/sensors/iai_kinect2.urdf.xacro]: Where is this file used? (iai_kinect2.urdf.xacro)

**Comment by UltronDestroyer** [comment]
Other than some comments above. Looks good to me. I'm going to test this and see if it works on my end.

File: pr2_description/robots/pr2.urdf.xacro (hunk 1 above)
Comment (archielee) [inline pr2_description/robots/pr2.urdf.xacro:27]: Yeah, in the original file there's a joint and link for the prosilica camera attached to the Kinect. I never removed from the include (edit: a word), just moved it inside an unless statement at the bottom for people using the original Kinect

File: pr2_description/urdf/sensors/iai_kinect2.urdf.xacro (hunk 3 above)
Comment (archielee) [inline pr2_description/urdf/sensors/iai_kinect2.urdf.xacro]: Oops that was for reference. Let me remove that.

File: pr2_description/robots/pr2.urdf.xacro (hunk 1 above)
Comment (UltronDestroyer) [inline pr2_description/robots/pr2.urdf.xacro:27]: Sounds good

**Comment by archielee** [comment]
Adding new upload_pr2.launch with optenvs

**Comment by UltronDestroyer** [comment]
Tested that this works on my end.

**Pull request merged**